packages/block-editor/src/components/iframe/index.js

```diff
@@ -419,6 +434,143 @@ function Iframe( {
 	// mode. They're only needed to capture focus in edit mode.
 	const shouldRenderFocusCaptureElements = tabIndex >= 0 && ! isPreviewMode;
 
+	const prevScaleRef = useRef( scaleValue );
+	const prevFrameSizeRef = useRef( frameSizeValue );
+	const prevContainerHeightRef = useRef( containerHeight );
+
+	// Scroll based on the new scale
+	useEffect( () => {
```

Comment:
Should/could this happen within the same useEffect as the useEffect that runs when toggling zoom out (that adds the animation classnames with handleZoomOutAnimationClassname)?